src/wasm-lib/kcl/tests/computed_var/ops.snap

```diff
@@ -7,9 +7,9 @@ description: Operations executed computed_var.kcl
     "type": "UserDefinedFunctionCall",
     "name": "cos",
     "functionSourceRange": [
-      1446,
-      1478,
-      2
+      0,
+      0,
+      0
```

Comment:
Interesting. These are seen as user-defined functions for operations and the Feature Tree.

Reply:
I guess because they're defined in KCL?

Reply:
Yes, we'll probably need to fix this. Or maybe it can actually point to the KCL stdlib? That would be cool if users could actually go to definition and see it. But I'm not worrying about this for right now.

Reply:
Yeah, the idea is they can click through to std, I think it'll take a little work to make that happen though